transform-service/latest/install/index.md

```diff
@@ -620,11 +622,11 @@ before continuing.
 6. Start the Transform Router Spring Boot app:
 
     ```java
-    java -DCORE_AIO_URL=http://localhost:8090
-     -DCORE_AIO_QUEUE=org.alfresco.transform.engine.aio.acs
-     -DACTIVEMQ_URL=failover:(tcp://server:61616)?timeout=3000
-     -DFILE_STORE_URL=http://localhost:8099/alfresco/api/-default-/private/sfs/versions/1/file
-     -jar alfresco-transform-router-3.0.x.jar
+    java -DCORE_AIO_QUEUE="org.alfresco.transform.engine.aio.acs" \
+         -DCORE_AIO_URL="http://localhost:8090" \
+         -DACTIVEMQ_URL="failover:(tcp://localhost:61616)?timeout=3000" \
```

Comment:
Above (lines 593, 594) the approach taken is taking "server" into brackets and replacing "localhost" to "<server>", which to my understanding is actually good and implies, that someone can host stuff in other places than localhost and should substitute the "<server>" with what they need there, but here the approach to exact same issue is different. I think this inconsistency should be fixed.

Reply:
As discussed, I'll address the consistency checks after all the releases are done. Thanks for highlighting the improvements 👍🏼